packages/client/src/config.schema.json

```diff
@@ -204,6 +204,18 @@
                         "networkConnectivityTimeout": {
                             "type": "number",
                             "default": 10000
+                        },
+                        "websocketServerEnableTls": {
+                            "type": "boolean",
+                            "default": true
+                        },
+                        "autoCertifierUrl": {
+                            "type": "string",
+                            "default": "https://ns1.fe6a54d8-8d6f-4743-890d-e9ecd680a4c7.xyz:59833"
+                        },
+                        "autoCertifiedSubdomainFilePath": {
```

Comment:
This is a file to specify the certificate (the subdomain is less important detail). The config option and the file name should reflect this.